build_runner/lib/src/server/server.dart

```diff
@@ -119,11 +132,31 @@ class ServeHandler implements BuildState {
 
   shelf.Response _performanceHandler(shelf.Request request) {
     var hideSkipped = false;
+    var detailedSlices = false;
+    var slicesResolution = 5;
+    var sortOrder = PerfSortOrder.startTimeAsc;
+    String filter;
```

Comment:
[nit] does `request.url.queryParameters['filter']` return null if `containsKey('filter')` returns `false`? If so, should we initialize this here?

Reply:
`info: Don't explicitly initialize variables to null`

Reply:
what I mean is `var filter = request.url.queryParameters['filter'];`.

Reply:
Ah... sure :)